style/design.css

```diff
@@ -111,229 +135,126 @@ label {
 
 .navbar-brand > img {
 	display: inline-block;
-	height: 25px;
+	height: 120px;
+	vertical-align: text-top;
+	
 }
 
-/* ===> Body <=== */
-
-#content {
-	background-color: #fff;
-	border: 1px solid #000;
-	margin: 40px auto;
+.shrink{
+	height:25px !important;
+	margin-right: 10px;
 }
 
-/* ===> Footer <=== */
-
-.footer {
-	border-top: 1px solid black;
-	padding: 20px;
-	font-size: 14px;
+.notshrink{
+	filter: drop-shadow(2px 5px 3px #999);
```

Comment:
I don't know why the -webkit- prefix disappeared here, I never said it wasn't necessary...
(Using an autoprefixer would avoid this kind of problem)